MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSCrashesUtil.m

```diff
@@ -72,4 +73,31 @@ + (NSURL *)logBufferDir {
   return logBufferDir;
 }
 
++ (NSURL *)wrapperExceptionsDir {
+  static NSURL *wrapperExceptionsDir = nil;
+  static dispatch_once_t predSettingsDir;
+
+  dispatch_once(&predSettingsDir, ^{
+    NSError *error = nil;
+    NSFileManager *fileManager = [[NSFileManager alloc] init];
+
+    // temporary directory for crashes grabbed from PLCrashReporter
+    NSURL *cachesDirectory = [[fileManager URLsForDirectory:NSCachesDirectory inDomains:NSUserDomainMask] lastObject];
```

Comment:
@jaelim-ms Not really; I tried to be consistent with the other methods because they are exactly identical. Perhaps this should be a helper function, though, to avoid duplication that exists. Thoughts?